language-translator/src/main/java/com/ibm/watson/language_translator/v3/model/ListLanguagesOptions.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.language_translator.v3.model;
+
+import com.ibm.cloud.sdk.core.service.model.GenericModel;
+
+/** The listLanguages options. */
+public class ListLanguagesOptions extends GenericModel {}
```

Comment:
is this supposed to be an empty Class? I think we need to copy what we have for v2. Check the method to make sure it works when you use this.

Reply:
It has no builder, so I just did this on my test
```
@Test
  public void testListLanguages_Success() {
    Languages response = service.listLanguages().execute().getResult();

    assertNotNull(response);
    assertNotNull(response.getLanguages());
    assertTrue(response.getLanguages().size() > 0);
  }
```
but with the builder it would look like this
```
@Test
  public void testListLanguages_Success() {
    ListLanguagesOptions listLanguagesOptions = 
            new ListLanguagesOptions.Builder()
            .build();
    Languages response = service.listLanguages(listLanguagesOptions).execute().getResult();

    assertNotNull(response);
    assertNotNull(response.getLanguages());
    assertTrue(response.getLanguages().size() > 0);
  }
```
I don't see this class in v2. With the builder it looks consistent with other models

Reply:
fixed in [29f598fca6c1b9cdb8c25f32d2f2f39392e0a25b](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/29f598fca6c1b9cdb8c25f32d2f2f39392e0a25b)